rest_framework_json_api/utils.py

```diff
@@ -72,7 +74,8 @@ def get_resource_name(context):
                 # The resource name is not a string - return as is
                 return resource_name
 
-            # the name was calculated automatically from the view > pluralize and format
+            # the name was calculated automatically from the view > pluralize
```

Comment:
Changing all these things is not helpful, even if your dev software did it for you. It only adds noise to the PR.